spec/support/app_actions.rb

```diff
@@ -77,8 +77,11 @@ def doctype_summary_results
   end
 
   def user_fills_out_time_constraints(details)
-    fill_in "Deadline (if you have one)", with: details[:needed_by_date]
+    fill_in "Deadline", with: details[:needed_by_date]
+    fill_in "Time this must be published by", with: details[:needed_by_time]
     fill_in "Must not be published before", with: details[:not_before_date]
+    choose "No"
```

Comment:
This should be removed from the `Add time input fields and update date labels` commit (I think it refers to the "Is this a legal deadline?" question, which hasn't been added yet.